site/reference/TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "command name"
```

Comment:
`cmd` per style guide? Can you update the ones below Pkg -> `pkg`.

Reply:
Updated the template and the titles for the pkg commands.